.github/workflows/code-climate-upload.yaml

```diff
@@ -0,0 +1,89 @@
+# Runs API unit tests and uploads them to Code Climate
+name: Code Climate API Unit Test Upload
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+    paths:
+      - "api/**"
+      - ".github/workflows/code-climate-upload.yaml"
+  workflow_dispatch:
+
+# ENV values match local development seed values only.
+# Does not allow for Keycloak or GC Notify tests
+env:
+  TESTING: true
+  API_PORT: 3004
+  MONGO_PORT: 27017
+  MONGO_PASSWORD: purchase-admin
```

Comment:
How is mongo used in the tests? This workflow isn't even starting the database

Reply:
It's not at the moment. I included this just in case future unit tests used these environment variables somewhere, to prevent them from being undefined.

Reply:
@dbarkowsky I'd say if you're not using them then remove them. It just adds confusion.

Reply:
Removed